wiktextract/extractor/de/page.py

```diff
@@ -0,0 +1,508 @@
+    "Ido Substantiv Übersicht",
+    "Irisch Adjektiv Übersicht",
+    "Irisch Substantiv Übersicht",
+    "Isländisch Name Übersicht",
+    "Isländisch Substantiv Übersicht",
+    "Isländisch Verb Übersicht",
+    "Italienisch Adjektiv Übersicht",
+    "Italienisch Substantiv Übersicht",
+    "Italienisch Verb Übersicht",
+    "Katalanisch Adjektiv Übersicht",
+    "Katalanisch Substantiv Übersicht",
+    "Katalanisch Verb Übersicht",
+    "Keilschrift Übersicht",
+    "Korsisch Substantiv Übersicht",
+    "Kroatisch Adjektiv Übersicht",
+    "Kroatisch Substantiv Übersicht",
+    "Kurdisch Substantiv Übersicht",
+    "Latein Adjektiv Übersicht",
+    "Latein Adverb Übersicht",
+    "Latein Substantiv Übersicht",
+    "Lettisch Substantiv Übersicht",
+    "Lettisch Verb Übersicht",
+    "Mazedonisch Substantiv Übersicht",
+    "Nahuatl Substantiv Übersicht",
+    "Neugriechisch Substantiv Übersicht",
+    "Niederdeutsch Adjektiv Übersicht",
+    "Niederländisch Adjektiv Übersicht" "Niederländisch Substantiv Übersicht",
+    "Niedersorbisch Substantiv Übersicht",
+    "Norwegisch Adjektiv Übersicht",
+    "Norwegisch Eigenname Übersicht",
+    "Norwegisch Substantiv Übersicht",
+    "Norwegisch Verb Übersicht",
+    "Okzitanisch Substantiv Übersicht",
+    "Okzitanisch Verb Übersicht",
+    "Papiamentu Substantiv Übersicht",
+    "Polnisch Grundzahl Übersicht",
+    "Polnisch Substantiv Übersicht",
+    "Portugiesisch Substantiv Übersicht",
+    "Rumänisch Numerale Übersicht",
+    "Rumänisch Personalpronomen Übersicht",
+    "Rumänisch Substantiv Übersicht",
+    "Rumänisch Verb Übersicht",
+    "Russisch Substantiv Übersicht",
+    "Sardisch Substantiv Übersicht",
+    "Schwedisch Adjektiv Übersicht",
+    "Schwedisch Adverb Übersicht",
+    "Schwedisch Eigenname Übersicht",
+    "Schwedisch Pronomen Übersicht",
+    "Schwedisch Verb Übersicht",
+    "Scots Substantiv Übersicht",
+    "Serbisch Substantiv Übersicht",
+    "Sesotho Substantiv Übersicht",
+    "Slowakisch Adjektiv Übersicht",
+    "Slowenisch Substantiv Übersicht",
+    "Spanisch Adjektiv Übersicht",
+    "Spanisch Substantiv Übersicht",
+    "Spanisch Verb Übersicht",
+    "Suaheli Substantiv Übersicht",
+    "Suaheli Verb Übersicht",
+    "Symbol Übersicht",
+    "Tschechisch universal Übersicht",
+    "Türkisch Substantiv Übersicht",
+    "Ukrainisch Substantiv Übersicht",
+    "Umbrisch Substantiv Übersicht",
+    "Ungarisch Adjektiv Übersicht",
+    "Ungarisch Substantiv Übersicht",
+    "Ungarisch Verb Übersicht",
+    "Usbekisch Substantiv Übersicht",
+    "Venezianisch Substantiv Übersicht",
+    "Walisisch Substantiv Übersicht",
+    "Westflämisch Substantiv Übersicht",
+}
+
+# Templates that should not be pre-expanded
+DO_NOT_PRE_EXPAND_TEMPLATES = {
+    "Ü-Tabelle",  # Translation table
+    "Quellen",  # Can be ignored since we have the <ref> tags in the tree
+}
+
+DO_NOT_PRE_EXPAND_TEMPLATES.update(FORM_TABLE_TEMPLATES)
+
+
+def fix_level_hierarchy_of_subsections(
```

Comment:
I think overwrite the subtitle template pages would be easier, like the [overrides/zh.json](https://github.com/tatuylonen/wiktextract/blob/master/overrides/zh.json) file

Reply:
wiktextract extractor code usually should not change the node tree structure, and I think for the German Wiktionary my above comment is simpler and more reliable. For example, we could override subtitle templates: `{"Vorlage:Beispiele": {"pre_expand": true, "body": "==== Beispiele ====", "namespace_id": 10}}`.